apps/client/src/hoc/withChecker.js

```diff
@@ -1,29 +1,27 @@
 import React, { useContext, useEffect, useState } from "react";
 
+import { Flow } from "../config";
 import { autofillMap, autofillResolvers } from "../config/autofill";
 import { CheckerContext, SessionContext } from "../context";
 import ErrorPage from "../pages/ErrorPage";
 import LoadingPage from "../pages/LoadingPage";
 import getChecker from "../sttr_client";
 import withTopic from "./withTopic";
 
-const dir =
-  process.env.REACT_APP_STTR_ENV === "production" ? "prod" : "staging";
-
 const withChecker = (Component) =>
   withTopic((props) => {
     const sessionContext = useContext(SessionContext);
     const checkerContext = useContext(CheckerContext);
     const [checker, setChecker] = useState(checkerContext.checker);
     const [error, setError] = useState();
     const { topic } = props;
-    const { sttrFile, slug } = topic;
+    const { flow, slug } = topic;
     const address = sessionContext[topic.slug]?.address;
 
-    if (sttrFile) {
+    if (flow !== Flow.olo) {
       useEffect(() => {
         if (!checker && !error) {
-          fetch(`${window.location.origin}/sttr/${dir}/${sttrFile}`)
+          fetch(`${window.location.origin}/sttr/sttr/${slug}.json`)
```

Comment:
sttr/sttr?

Reply:
yes, good point. I understand this is confusing. Will fix.